From 63b49e5c5b64ebfeb5d33ab8e3092b36a22dc0b9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Vincent=20Stehl=C3=A9?= <vincent.stehle@laposte.net>
Date: Wed, 23 Nov 2016 09:59:29 +0100
Subject: [PATCH] Align encoding
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Vincent Stehlé <vincent.stehle@laposte.net>
---
 riscv/encoding.h | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/riscv/encoding.h b/riscv/encoding.h
index 35e0f9f..6100e72 100644
--- a/riscv/encoding.h
+++ b/riscv/encoding.h
@@ -714,7 +714,7 @@
 #define CSR_SIP 0x144
 #define CSR_SPTBR 0x180
 #define CSR_MSTATUS 0x300
-#define CSR_MISA 0x301
+#define CSR_MISA 0xf10
 #define CSR_MEDELEG 0x302
 #define CSR_MIDELEG 0x303
 #define CSR_MIE 0x304
@@ -731,8 +731,8 @@
 #define CSR_DCSR 0x7b0
 #define CSR_DPC 0x7b1
 #define CSR_DSCRATCH 0x7b2
-#define CSR_MCYCLE 0xb00
-#define CSR_MINSTRET 0xb02
+#define CSR_MCYCLE 0xf00
+#define CSR_MINSTRET 0xf02
 #define CSR_MHPMCOUNTER3 0xb03
 #define CSR_MHPMCOUNTER4 0xb04
 #define CSR_MHPMCOUNTER5 0xb05
@@ -762,8 +762,8 @@
 #define CSR_MHPMCOUNTER29 0xb1d
 #define CSR_MHPMCOUNTER30 0xb1e
 #define CSR_MHPMCOUNTER31 0xb1f
-#define CSR_MUCOUNTEREN 0x320
-#define CSR_MSCOUNTEREN 0x321
+#define CSR_MUCOUNTEREN 0x310
+#define CSR_MSCOUNTEREN 0x311
 #define CSR_MHPMEVENT3 0x323
 #define CSR_MHPMEVENT4 0x324
 #define CSR_MHPMEVENT5 0x325
@@ -829,8 +829,8 @@
 #define CSR_HPMCOUNTER29H 0xc9d
 #define CSR_HPMCOUNTER30H 0xc9e
 #define CSR_HPMCOUNTER31H 0xc9f
-#define CSR_MCYCLEH 0xb80
-#define CSR_MINSTRETH 0xb82
+#define CSR_MCYCLEH 0xf80
+#define CSR_MINSTRETH 0xf82
 #define CSR_MHPMCOUNTER3H 0xb83
 #define CSR_MHPMCOUNTER4H 0xb84
 #define CSR_MHPMCOUNTER5H 0xb85
-- 
2.10.2

