From 70d59d3984ad49f92afcc9ef95db78f86d2e020a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Vincent=20Stehl=C3=A9?= <vincent.stehle@laposte.net>
Date: Mon, 14 Nov 2016 15:47:36 +0100
Subject: [PATCH] riscv: remove -msoft-float
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Vincent Stehlé <vincent.stehle@laposte.net>
---
 arch/riscv/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/riscv/Makefile b/arch/riscv/Makefile
index 368f264..3cf9e07 100644
--- a/arch/riscv/Makefile
+++ b/arch/riscv/Makefile
@@ -45,8 +45,6 @@ KBUILD_CFLAGS += -mno-atomic
 endif
 
 KBUILD_CFLAGS += -Wall
-KBUILD_CFLAGS += -msoft-float
-KBUILD_AFLAGS += -msoft-float
 
 ifeq ($(CONFIG_RVC),y)
 	KBUILD_CFLAGS += -mrvc
-- 
2.10.2

